connector/src/main/java/org/geysermc/connector/network/session/cache/BookEditCache.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2019-2020 GeyserMC. http://geysermc.org
```

Comment:
Needs updating to 2021
```suggestion
 * Copyright (c) 2019-2021 GeyserMC. http://geysermc.org
```